core/actionProxy/README.md

```diff
@@ -12,17 +12,21 @@ The script `buildAndPush.sh` is provided for your convenience. The following com
 runs the included example Docker action container using OpenWhisk.
 
 ```
-# install ./dockerSkeleton with example
-> wsk sdk install docker
+# install dockerSkeleton with example
+wsk sdk install docker
+
 # change working directory
-> cd dockerSkeleton
+cd dockerSkeleton
+
 # build/push, argument is your docker hub user name and a valid docker image name
-> ./buildAndPush <dockerhub username>/whiskexample
+./buildAndPush <dockerhub username>/whiskexample
+
 # create docker action
-> wsk action create dockerSkeletonExample --docker <dockerhub username>/whiskExample
+wsk action create dockerSkeletonExample --docker <dockerhub username>/whiskExample
+
 # invoke created action
-> wsk action invoke dockerSkeletonExample --blocking
+wsk action invoke dockerSkeletonExample --blocking
 ```
 
-The binary to execute must be located in the `client` folder. The name of the binary
-(or shell script as in the standard bundle) must be `action`.
+The binary to execute must be located in the `/action` folder. The name of the binary
+(or shell script as in the standard bundle) must be `/action/a.out`.
```

Comment:
To my brain `a.out` means "binary output from a compiler". This name may be confusing despite the comment saying that shell scripts are also allowed. Perhaps a more generic name like `/action/exec` would help?

Also, wouldn't any executable file be acceptable? It seems like calling out only shell scripts may seem to exclude things like Python scripts.

How about:
> The executable file must be located in the `/acton` folder. The name of the executable must be `/action/exec` and can be any file with executable permissions.